Storage/test.cc

```diff
@@ -2632,13 +2645,13 @@ CURRENT_STRUCT(CQSCommand) {
   // Keep these two lines together.
   static int DoThrowCurrentExceptionLine() { return __LINE__ + 1; }
   static void DoThrowCurrentException() { CURRENT_THROW(CQSTestException()); }
-  // clang-format on
 
   template <typename MUTABLE_FIELDS>
-  Response Command(MUTABLE_FIELDS & fields, const std::string& restful_url_prefix) const {
+  Response Command(MUTABLE_FIELDS& fields, const current::storage::rest::cqs::CQSParameters& cqs_parameters) const {
+    // clang-format on
```

Comment:
Missing `// clang-format off` later on?

Reply:
Err, I mean -- move this `// clang-format on` two lines above, before the member function declaration?